sw/device/tests/hmac_smoketest.c

```diff
@@ -113,11 +113,11 @@ static void push_message(const dif_hmac_t *hmac, const char *data, size_t len) {
 static void wait_for_fifo_empty(const dif_hmac_t *hmac) {
   uint32_t fifo_depth;
   do {
-    dif_hmac_result_t res = dif_hmac_fifo_count_entries(hmac, &fifo_depth);
+    dif_result_t res = dif_hmac_fifo_count_entries(hmac, &fifo_depth);
 
-    CHECK(res != kDifHmacBadArg,
+    CHECK(res != kDifBadArg,
           "Invalid arguments encountered checking FIFO depth.");
-    CHECK(res == kDifHmacOk, "Unknown error encountered checking FIFO depth.");
+    CHECK(res == kDifOk, "Unknown error encountered checking FIFO depth.");
```

Comment:
It seems that we can now start thinking about adding `CHECK_DIF_OK` or `CHECK_OK`.

Reply:
yes, definitely, great idea! added item to track this in #8142, will add this in a future PR.

Reply:
We've talked about [adding](https://github.com/lowRISC/opentitan/pull/2539/files#r442263187) `CHECK_OK` a few times over the years it seems :)